src/components/number/number.css

```diff
@@ -0,0 +1,41 @@
+@import '../../styles/colors-v2.css';
+@import "../../styles/spacing.css";
+@import "../../styles/typography.css";
+
+.input {
```

Comment:
Do we want to compose from the existing styling in `Input`? 

I think the only difference is the padding? I'm not even sure tbh.

Reply:
It's exactly the same styling. Should I pull it back into a separate shared styling sheet?

Reply:
No, just compose from it.

Reply:
@cincospenguinos  ?